bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/cbemmBaseOptionElectricityMeter.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="cbemm_base_electricitymeter" listed="false">
```

Comment:
You could reference the allowed bridge types with `<supported-bridge-type-refs>`.